mllib/src/main/scala/org/apache/spark/ml/util/SchemaUtils.scala

```diff
@@ -34,10 +34,11 @@ object SchemaUtils {
    * @param colName  column name
    * @param dataType  required column data type
    */
-  def checkColumnType(schema: StructType, colName: String, dataType: DataType): Unit = {
+  def checkColumnType(schema: StructType, colName: String, dataType: DataType,
```

Comment:
style: Please make sure you follow the style guide linked from [https://cwiki.apache.org/confluence/display/SPARK/Contributing+to+Spark]

If this fits on 1 line (<= 100 chars), then it should.  If not, then there should be 1 param per line.